src/DesignKit.tsx

```diff
@@ -4636,6 +4647,384 @@ const AlertSection: React.FC = () => {
+            'lastLogin',
+            'Last Login',
+            columnWidths.lastLogin,
+            'lastLogin',
+          );
+        }
+        if (columnId === 'state') {
+          return {
+            id: 'state',
+            renderer: (record: Person) => ({
+              allowAdd: false,
+              allowOverlay: true,
+              copyData: record.state.toLocaleLowerCase(),
+              data: {
+                appTheme: theme,
+                kind: STATE_CELL,
+                state: record.state,
+              },
+              kind: GridCellKind.Custom,
+            }),
+            themeOverride: { cellHorizontalPadding: 13 },
+            title: 'State',
+            tooltip: (record: Person) => record.state.toLocaleLowerCase(),
+            width: columnWidths.state,
+          };
+        }
+        return defaultSelectionColumn<Person>(selection.rows, false);
+      }),
+    [columnWidths, columnsOrder, selection, theme],
+  );
+
+  const [page, setPage] = useState(0);
+  const [gridData, setGridData] = useState<Loadable<Person>[]>([]);
+  const PAGE_SIZE = 8;
+  const TOTAL = 64; // simulating API call with 64 total items across all pages
+
+  useEffect(() => {
+    // simulate API call on page update:
+    const data: Loadable<Person>[] = [
+      Loaded({
+        lastLogin: new Date('01/01/2011'),
+        name: 'Alice',
+        score: 99,
+        state: State.ERROR,
+      }),
+      Loaded({ lastLogin: new Date('02/02/2012'), name: 'Bob', score: 98, state: State.PAUSED }),
+      Loaded({
+        lastLogin: new Date('03/03/2013'),
+        name: 'Charlie',
+        score: 97,
+        state: State.STOPPED,
+      }),
+      Loaded({
+        lastLogin: new Date('04/04/2014'),
+        name: 'David',
+        score: 96,
+        state: State.SUCCESS,
+      }),
+      Loaded({ lastLogin: new Date('05/05/2015'), name: 'Eve', score: 95, state: State.ERROR }),
+      Loaded({
+        lastLogin: new Date('06/06/2016'),
+        name: 'Frank',
+        score: 94,
+        state: State.PAUSED,
+      }),
+      Loaded({
+        lastLogin: new Date('07/07/2017'),
+        name: 'Grace',
+        score: 93,
+        state: State.STOPPED,
+      }),
+      Loaded({
+        lastLogin: new Date('08/08/2018'),
+        name: 'Heidi',
+        score: 92,
+        state: State.SUCCESS,
+      }),
+    ];
+    const pagedData: Loadable<Person>[] = [];
+    let limit = page;
+    let end = false;
+    while (limit >= 0 && !end) {
+      pagedData.push(...data);
+      limit--;
+      if (pagedData.length === TOTAL) end = true;
+    }
+    setTimeout(() => {
+      setGridData(pagedData);
+    }, 1000);
+  }, [page]);
+
+  return (
+    <ComponentSection id="DataGrid">
+      <SurfaceCard>
+        <p>
+          <code>DataGrid</code> is used to display tabular data in a grid, and is implemented with
+          the <a href="https://docs.grid.glideapps.com/">Glide Data Grid library</a>.
```

Comment:
super nit: the `<p>` tag should only be around this sentence, not the full card.